<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductGuiWidget</class>
 <widget class="QWidget" name="ProductGuiWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>363</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>242</width>
    <height>363</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>242</width>
    <height>363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>5</y>
     <width>234</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
margin: 8px 3px;
/*border-radius: 10px;*/
/*border: 3px solid #22a9f7;*/
/*background-color: red;*/
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="lb_photo">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>11</y>
      <width>221</width>
      <height>320</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
border: 0;
margin: 0;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/img/assets/img/Products/100.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>242</y>
      <width>226</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
margin: 0;
border-radius: 15px;
/*border: 3px solid #22a9f7;*/
/*background-color: #bfbfbf*/;
	background-color: rgba(191, 191, 191, 200);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_oldPrice">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>74</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
       <bold>false</bold>
       <strikeout>true</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
border: 0;
border-radius: 10px;
margin: 0;
background-color: rgba(0,0,0,0);
}</string>
     </property>
     <property name="text">
      <string>EGP 220.00 </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_rate_0_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>77</y>
       <width>171</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>15</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
background-color: #ffffff;
border-radius: 7px;   
border: 0;
margin: 0;           
}
QProgressBar::chunk {
border-radius: 7px;
	background-color: #00BD0D;
}</string>
     </property>
     <property name="value">
      <number>63</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_name">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>35</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
border: 0;
margin: 0;
background-color: rgba(0,0,0,0);
}
*:hover {
font: 600 14pt &quot;Segoe UI&quot;;
}</string>
     </property>
     <property name="text">
      <string>T-Shirt - White</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_price">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>9</y>
       <width>116</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">*{
/*border: 0;*/
border-radius: 10px;
margin: 0;
background-color: rgba(0,0,0,0);
}</string>
     </property>
     <property name="text">
      <string>EGP 176.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lb_offer">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
background-color: #feefde;
border-radius: 15px;
border: 0;
margin: 0;
color: #f68d49;
}</string>
    </property>
    <property name="text">
     <string>-20%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_availability">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>50</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
 color: #b32400;
	background-color: rgba(255, 255, 255, 0);
 }
 </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/img/assets/img/icons/out-of-stock-icon-1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
